src/main/java/core/basesyntax/repoter/ReportGenerator.java

```diff
@@ -0,0 +1,7 @@
+package core.basesyntax.repoter;
+
+import java.util.Map;
+
+public interface ReportGenerator {
```

Comment:
Consider renaming the 'ReportGenerator' interface to something more specific to its functionality, such as 'FruitReportGenerator', to improve readability and maintainability of the code.